ontotext-yasgui-web-component/src/services/utils/html-builder.ts

```diff
@@ -0,0 +1,92 @@
+export class HtmlBuilder {
+
+  private html = '';
+
+  append(content: string): HtmlBuilder {
+    this.html += content;
+    return this;
+  }
+
+  openDiv(divClass: string): HtmlBuilder {
+    return this.append("<div")
+      .addClass(divClass)
+      .append(">");
+  }
+
+  closeDiv(): HtmlBuilder {
+    return this.append("</div>");
+  }
+
+  openUl(ulClass: string): HtmlBuilder {
+    return this.append(`<ul class="${ulClass}">`);
+  }
+
+  closeUl(): HtmlBuilder {
+    return this.append('</ul>');
+  }
+
+  addLi(liContent: string): HtmlBuilder {
+    return this.append(`<li>${liContent}</li>`);
+  }
+
+  addLink(title: string, linkClass: string, href: string, content = ''): HtmlBuilder {
+    return this.append("<a")
+      .addTitle(title)
+      .addClass(linkClass)
+      .addHref(href)
+      .append(">")
+      .addContent(content)
+      .append("</a>");
+  }
+
+  addP(content: string, classes: string): HtmlBuilder {
+    return this.append('<p')
+      .addClass(classes)
+      .append('>')
+      .append(content)
+      .append('</p>');
+  }
+
+  addCopyResourceLinkButton(uri: string, classes = ''): HtmlBuilder {
+    return this.append("<copy-resource-link-button")
```

Comment:
This note is not for this line but I now saw that when I click on a copy link , the page jumpt to top which is somehow annoying.